### PR TITLE
Removed unnecessary replacing of variable from parent

### PR DESCRIPTION
**Pull request by Sekiphp**
### Description (*)

`Mage_Directory_Model_Resource_Country_Collection` extending `Varien_Data_Collection` where is `$_items` already defined:
```
    /**
     * Collection items
     *
     * @var array
     */
    protected $_items = array();

```
And there was diferrent initial value.

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All automated tests passed successfully (all builds are green)

### CODE REVIEW

File: app/code/core/Mage/Directory/Model/Resource/Country/Collection.php [hunk 1]
```diff
@@ -33,11 +33,6 @@
  */
 class Mage_Directory_Model_Resource_Country_Collection extends Mage_Core_Model_Resource_Db_Collection_Abstract
 {
-    /**
-     * @var Mage_Directory_Model_Country[]
-     */
-    protected $_items;
```
Comment (midlan) [inline app/code/core/Mage/Directory/Model/Resource/Country/Collection.php:39]: podle mě to tu bylo schválně kvůli phpdoc
dokážeš tam dostat phpdoc bez přepsání tý proměnný?

Comment (midlan) [inline app/code/core/Mage/Directory/Model/Resource/Country/Collection.php:39]: We will loose phpdoc (typehint) by this?

**Review by midlan — APPROVED** [review]

File: app/code/core/Mage/Directory/Model/Resource/Country/Collection.php (hunk 1 above)
Comment (midlan) [reply app/code/core/Mage/Directory/Model/Resource/Country/Collection.php:39]: 👍  resolved in ea13c48a0bbe48dfdc0a786aa0e9ea8decec7173

**Review by colinmollenhour — APPROVED** [review]

**Comment by sreichel** [comment]
> The last one in particular is helpful when iterating a collection so that the loop value has correct type hinting automatically, at least in PhpStorm.

Which phpStorm version do you use? I thought this would already work? 

Wait ... no time to check, but I solved in another way. I added something like `@method SomeClass[] getCollection()`, but not for all classes yet. See #784. I'd prefer my way, because all methods are covered. (if i'm right here)

**Comment by colinmollenhour** [comment]
PhpStorm is smart enough to use the return type of `getIterator()` inside a loop to determine the iterator item type because `Varien_Data_Collection` implements `IteratorAggregate`. So if `getIterator()` hints `Mage_Directory_Model_Country[]` then this will work with no additional hints (`$country` will be detected as an instance of `Mage_Directory_Model_Country`):

```php
foreach (Mage::getResourceModel('directory/country_collection') as $country) {
    $country->getIso2Code(); // returns string
}
```

Also this:

```php
Mage::getResourceModel('directory/country_collection')->getFirstItem()->getIso2Code(); // returns string
```


My example with `@method` annotations is the only way I know to get this to work.

**Review by sreichel — APPROVED** [review]

**Comment by sreichel** [comment]
@colinmollenhour you're right. `getIterator()` is something i missed for all doc block update :/

**Review by LukasCesal — APPROVED** [review]

**Review by colinmollenhour — APPROVED** [review]

**Pull request merged**